codegen/src/main/java/software/amazon/awssdk/codegen/poet/auth/scheme/AuthSchemeCodegenMetadata.java

```diff
@@ -69,8 +101,16 @@ public static AuthSchemeCodegenMetadata fromAuthType(AuthType type) {
         switch (type) {
             case BEARER:
                 return BEARER;
-            default:
+            case NONE:
+                return NO_AUTH;
+            case V4:
+            case V4_UNSIGNED_BODY:
```

Comment:
I think this case may need more changes, but doesn't block this PR.

Reply:
Sure, can you please elaborate on which other changes are needed?

Reply:
We will handle in separate PR, when corresponding signer solution is determined, which is being discussed offline. No changes needed for this in this PR.